src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -1211,6 +1211,12 @@ public void increaseBoughtCitizenCost()
 
     }
 
+    @Override
+    public Set<EntityPlayer> getImportantMessageEntityPlayers()
+    {
+        return null;
```

Comment:
Please do not return `null` here but just as in the other method an empty `set`?